encodings/runend/src/default/compute.rs

```diff
@@ -8,7 +8,7 @@ use vortex_dtype::match_each_integer_ptype;
 use vortex_error::VortexResult;
 use vortex_scalar::Scalar;
 
-use crate::RunEndArray;
+use crate::default::RunEndArray;
```

Comment:
I'm not a fan of non-descriptive names like "default", was this a relic of a refactor?

Reply:
Would you prefer its top level. I could go for equality?